### PR TITLE
chore: Translations for login

### PR DESCRIPTION
**Pull request by ui-okta**
## Description
Auto copy of latest translations by okta-i18n script.
### Bacon task details
- Task ID: a98eba36d3924d5db083369699244f50 
- Requested by: yasmin.vanya@okta.com
- Resolves: [OKTA-CHANGE-ME](https://oktainc.atlassian.net/browse/OKTA-CHANGE-ME)
## Notification sent to
@okta/eng-globalizationcore @haishengwu-okta @aarongranick-okta

### CODE REVIEW

**Review by glenfannin-okta — APPROVED** [review]

**Review by leemchale-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login_sv.properties [hunk 1]
```diff
@@ -900,6 +901,8 @@ oie.authenticator.enroll.error.fail = Det g\u00E5r inte att registrera autentise
 oie.authenticator.verify.error.fail = Det g\u00E5r inte att verifiera autentiseraren. F\u00F6rs\u00F6k igen.
 oie.configuration.error = N\u00E5gonting gick fel. Eventuell felkonfigurering detekterad. Kontakta support.
 oie.auth.logo.aria.label = Autentiseringslogotyp
+oie.challenge.answer.showAnswer = Visa svaret
+oie.challenge.answer.hideAnswer = Hide answer
```
Comment (mauriciocastillosilva-okta) [inline packages/@okta/i18n/src/properties/login_sv.properties:905]: Translations on this file are in English, which is wrong. Let me figure out what happened

**Review by mauriciocastillosilva-okta — COMMENTED** [review]
Just one comment, I'll find out about what is going on

File: packages/@okta/i18n/src/properties/login_sv.properties (hunk 1 above)
Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login_sv.properties:905]: We saw this for other *_sv files as well this month

**Comment by mauriciocastillosilva-okta** [comment]
Closing this one in favor of https://github.com/okta/okta-signin-widget/pull/3172 which has the fixes for the missing translations

**Pull request closed without merging**